substrate/frame/treasury/src/benchmarking.rs

```diff
@@ -215,6 +215,8 @@ mod benchmarks {
 		Ok(())
 	}
 
+	// This benchmark is short-circuited if `SpendOrigin` cannot provide
```

Comment:
```suggestion
	/// This benchmark is short-circuited if `SpendOrigin` cannot provide
```
Should be possible to use real doc comments here.